resources/keywords/provision_dbinstance.resource

```diff
@@ -43,6 +57,7 @@ User Enters Data To Create Database Instance On Admin View
     Element Should Be Visible    ${txtCreatingDBInst}
 
 User Enters Data To Create Instance On Dev View
+    User Selects DBProvider And Provider Account
```

Comment:
@ikanias On developer view, the Database provider type would be auto populated, So this keyword might not be valid here